gormschema/gorm.go

```diff
@@ -24,8 +25,9 @@ func New(dialect string, opts ...Option) *Loader {
 type (
 	// Loader is a Loader for gorm schema.
 	Loader struct {
-		dialect string
-		config  *gorm.Config
+		dialect                           string
+		createConstraintsAfterCreateTable bool
```

Comment:
change it to `detachFKs`/`deferredFKs` - anything that looks like the current naming